src/routes/courses.js

```diff
@@ -57,6 +57,8 @@ router.post('/search', async (req, res) => {
     }
 });
 
+const max_search_size = 15000;

```

Comment:
Suggest to make this constant a environment variable (i.e. `COURSE_REQUEST_LIMIT`) for better maintenance flexibility 🥰

Reply:
Sure! That'll be better.